sdk/standbypool/arm-standbypool/CHANGELOG.md

```diff
@@ -1,15 +1,40 @@
 # Release History
-
-## 1.0.2 (Unreleased)
-
+    
+## 2.0.0 (2025-04-16)
+    
 ### Features Added
 
-### Breaking Changes
+  - Added Interface ErrorAdditionalInfo
+  - Added Interface ErrorDetail
+  - Added Interface ErrorResponse
+  - Added Interface PoolContainerGroupStateCount
+  - Added Interface PoolStatus
+  - Added Interface PoolVirtualMachineStateCount
+  - Added Interface StandbyContainerGroupPoolForecastValues
+  - Added Interface StandbyContainerGroupPoolPrediction
+  - Added Interface StandbyVirtualMachinePoolForecastValues
+  - Added Interface StandbyVirtualMachinePoolPrediction
+  - Added Type Alias HealthStateCode
+  - Added Type Alias PoolContainerGroupState
+  - Added Type Alias PoolVirtualMachineState
+  - Interface ContainerGroupInstanceCountSummary has a new optional parameter zone
+  - Interface StandbyContainerGroupPoolResourceProperties has a new optional parameter zones
+  - Interface StandbyContainerGroupPoolResourceUpdateProperties has a new optional parameter zones
+  - Interface StandbyContainerGroupPoolRuntimeViewResourceProperties has a new optional parameter prediction
+  - Interface StandbyContainerGroupPoolRuntimeViewResourceProperties has a new optional parameter status
+  - Interface StandbyVirtualMachinePoolRuntimeViewResourceProperties has a new optional parameter prediction
+  - Interface StandbyVirtualMachinePoolRuntimeViewResourceProperties has a new optional parameter status
+  - Added Enum KnownHealthStateCode
+  - Added Enum KnownPoolContainerGroupState
+  - Added Enum KnownPoolVirtualMachineState
+  - Added Enum KnownVersions
+  - Enum KnownVirtualMachineState has a new value Hibernated
 
-### Bugs Fixed
-
-### Other Changes
+### Breaking Changes
 
+  - Type of parameter instanceCountsByState of interface ContainerGroupInstanceCountSummary is changed from PoolResourceStateCount[] to PoolContainerGroupStateCount[]
```

Comment:
can you find out which spec pr introduce the breaking?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/32288